src/item.h

```diff
@@ -1299,6 +1299,9 @@ class item : public visitable<item>
          * Returns clothing layer for item which will always be 0 for non-wearable items.
          */
         int get_layer() const;
+
+        layer_level layer() const;
```

Comment:
This is just confusing. What is the difference between `get_layer()` and just `layer()`?

Hint: if you kept the `get_layer` function to keep its Lua binding: enumerations can be exported to Lua as well.

Reply:
I specifically *added* another getter (but didn't call it `get_...`) so it wouldn't get exported to lua. Simply because I have no fricken clue what's happening with the export, or ultimately in lua, and ultimately, that's completely external to the scope of my changes.

Reply:
Something like that in `class_definitions.lua`:

```LUA
enums = {
   layer_level = {
       "UNDERWEAR",
       "REGULAR_LAYER",
       "WAIST_LAYER",
       "OUTER_LAYER",
       "BELTED_LAYER"
   },
```

Reply:
Don't worry about LUA unless you have a need for it, just do what your code needs.